src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -230,23 +227,23 @@ public AzureAppConfigurationOptions UseFeatureFlags(Action<FeatureFlagOptions> c
             {
                 // Select clause is not present
                 options.FeatureFlagSelectors.Add(new KeyValueSelector
-                {
-                    KeyFilter = FeatureManagementConstants.FeatureFlagMarker + "*",
-                    LabelFilter = options.Label == null ? LabelFilter.Null : options.Label
-                });
+                (
+                    keyFilter: FeatureManagementConstants.FeatureFlagMarker + "*",
+                    labelFilter: options.Label == null ? LabelFilter.Null : options.Label
+                ));
             }
 
             foreach (var featureFlagSelector in options.FeatureFlagSelectors)
             {
                 var featureFlagFilter = featureFlagSelector.KeyFilter;
                 var labelFilter = featureFlagSelector.LabelFilter;
 
-                Select(featureFlagFilter, labelFilter);
+                Select(featureFlagFilter!, labelFilter!);
 
                 _multiKeyWatchers.AppendUnique(new KeyValueWatcher
                 {
-                    Key = featureFlagFilter,
-                    Label = labelFilter,
+                    Key = featureFlagFilter!,
+                    Label = labelFilter!,
```

Comment:
Here there's no check that `featureFlagSelector.KeyFilter` and `featureFlagSelector.LabelFilter` are not null. They might be since `featureFlagSelector` could be initialized with `SnapshotName`